spec/controllers/users/piv_cac_login_controller_spec.rb

```diff
@@ -4,16 +4,13 @@
   describe 'GET new' do
     before do
       stub_analytics
-      allow(@analytics).to receive(:track_event)
```

Comment:
ooo good call removing this so that the checker can enforce correctly

Reply:
Yeah this seems to be pretty common actually, we probably want to replace these with `stub_analytics` + `have_logged_event`.

Currently 71 results with the regex search: `\.to receive\(:track_event\)(?!\.)`